### PR TITLE
added layers and implemented DB lobic

### PR DESCRIPTION
**Pull request by RLudchenko**

### CODE REVIEW

File: src/main/java/car/project/service/impl/CarDoorServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import car.project.lib.Inject;
+import car.project.lib.Service;
+import car.project.model.CarDoor;
+import car.project.service.CarDoorService;
+import java.util.List;
+
+@Service
+public class CarDoorServiceImpl implements CarDoorService {
+    @Inject
+    CarDoorDao carDoorDao;
```
Comment (dmytro-hryhoruk) [inline src/main/java/car/project/service/impl/CarDoorServiceImpl.java]: access modifiers

File: src/main/resources/hibernate.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <property name="hibernate.connection.username">root</property>
+    <property name="hibernate.connection.password">1234</property>
+    <property name="hibernate.show_sql">true</property>
+    <property name="hibernate.hbm2ddl.auto">create-drop</property>
+
+    <mapping class="car.project.model.Car" />
+    <mapping class="car.project.model.CarDoor" />
+    <mapping class="car.project.model.CarWheel" />
+  </session-factory>
+</hibernate-configuration>
```
Comment (dmytro-hryhoruk) [inline src/main/resources/hibernate.cfg.xml]: empty line

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/resources/hibernate.cfg.xml (hunk 2 above)
Comment (RLudchenko) [reply src/main/resources/hibernate.cfg.xml]: Done

File: src/main/java/car/project/service/impl/CarDoorServiceImpl.java (hunk 1 above)
Comment (RLudchenko) [reply src/main/java/car/project/service/impl/CarDoorServiceImpl.java]: done

**Pull request merged**